ecosystem/sep-0010.md

```diff
@@ -14,6 +14,8 @@ Version 2.0.0
 
 This SEP defines the standard way for clients such as wallets or exchanges to create authenticated web sessions on behalf of a user who holds a Stellar account. A wallet may want to authenticate with any web service which requires a Stellar account ownership verification, for example, to upload KYC information to an anchor in an authenticated way as described in [SEP-12](sep-0012.md).
 
+This SEP also supports clients who control [pooled accounts](#pooled-accounts) containing funds owned by _many_ users. These clients manage their users' Stellar accounts and hold the secret keys for those accounts.
```

Comment:
Suggested change to align with the other comment about pooled accounts:
```suggestion
This SEP also supports authenticating clients who will prove a claim to a [memo](#memos) value of a Stellar account that is useful for authenticating clients who share a Stellar account.
```

Reply:
`who will prove a claim to a memo value` doesn't read well. What about:
```suggestion
This SEP also supports authenticating clients that share a Stellar account. Each unique client should be identified with a [memo](#memos) value included in the initial GET request as well as the JWT body.
```

Reply:
That's clearer. I think we should leave out technical details in this paragraph, and instead of stating what a client should do, we don't need to be prescriptive, and can highlight who it relates to. 
```suggestion
This SEP also supports authenticating clients that share a Stellar account and use [memos](#memos) to distinguish users or accounts for payments.
```